eras/conway/impl/testlib/Test/Cardano/Ledger/Conway/Binary/Regression.hs

```diff
@@ -19,7 +54,7 @@ spec ::
   Spec
 spec = describe "Regression" $ do
   it "DeserialiseFailure on resubmitting Conway Tx with invalid plutus script #4198" $ do
-    expectRightDeep_ $
+    Common.expectRightDeep_ $
```

Comment:
You can do this (or `it "foo" $ io $ do ...`), if you want to avoid importing `Common` as qualified 
```suggestion
    io $ expectRightDeep_ $
```